<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>664</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Param√®tres</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>381</width>
      <height>651</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::ExpandingFieldsGrow</enum>
       </property>
       <property name="rowWrapPolicy">
        <enum>QFormLayout::DontWrapRows</enum>
       </property>
       <property name="labelAlignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="horizontalSpacing">
        <number>-1</number>
       </property>
       <property name="verticalSpacing">
        <number>-1</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Listening Port</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="networkListeningPort">
         <property name="placeholderText">
          <string/>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Sending Port</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="networkSendingPort"/>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Server IP</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QLineEdit" name="networkServerIP"/>
       </item>
       <item row="14" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Pirate Boats Start Time</string>
         </property>
        </widget>
       </item>
       <item row="14" column="1">
        <widget class="QLineEdit" name="pirateBoatsStart"/>
       </item>
       <item row="18" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Pirate Boat Increase Rate</string>
         </property>
        </widget>
       </item>
       <item row="18" column="1">
        <widget class="QLineEdit" name="pirateBoatsIncreaseRate"/>
       </item>
       <item row="23" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Game Timer (s)</string>
         </property>
        </widget>
       </item>
       <item row="23" column="1">
        <widget class="QLineEdit" name="timerGame"/>
       </item>
       <item row="28" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Challenge Timer</string>
         </property>
        </widget>
       </item>
       <item row="28" column="1">
        <widget class="QLineEdit" name="timerChallenge"/>
       </item>
       <item row="33" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Disturbance Timer</string>
         </property>
        </widget>
       </item>
       <item row="33" column="1">
        <widget class="QLineEdit" name="timerDisturbance"/>
       </item>
       <item row="38" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Scores File</string>
         </property>
        </widget>
       </item>
       <item row="38" column="1">
        <widget class="QLineEdit" name="fileScores"/>
       </item>
       <item row="40" column="1">
        <widget class="QPushButton" name="applyButton">
         <property name="text">
          <string>Apply</string>
         </property>
        </widget>
       </item>
       <item row="41" column="1">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Clear Scores</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>applyButton</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>600</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>331</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
